api/observability/v1/output_types.go

```diff
@@ -601,7 +601,8 @@ type Kafka struct {
 	// 3. foo.{.bar.baz||.qux.quux.corge||.grault||"nil"}-waldo.fred{.plugh||"none"}
 	//
 	// +kubebuilder:validation:Pattern:=`^(([a-zA-Z0-9-_.\/])*(\{(\.[a-zA-Z0-9_]+|\."[^"]+")+((\|\|)(\.[a-zA-Z0-9_]+|\.?"[^"]+")+)*\|\|"[^"]*"\})*)*$`
-	Topic string `json:"topic"`
+	// +kubebuilder:validation:Optional
```

Comment:
My understanding is you need `omitempty` here according to Alan's explanation?

Reply:
added per suggestion of calvin... see line 605